GLOSSARY.md

```diff
@@ -40,7 +40,13 @@ This glossary accompanies the [GitOps Principles](./PRINCIPLES.md), and other su
     Git, from which GitOps derives its name, is the canonical example used as this state store but any other system that meets these criteria may be used.
     In all cases, these state stores must be properly configured and precautions must be taken to comply with requirements set out in the GitOps Principles.
 
+- ## Intermediate State Store
+    A system for storing a copy of the declarations that are mastered in the State Store. This system's purpose is intended to bridge the gap in availability between that of the State Store and the expected availability to make configuration changes to the Software System. The Intermediate State Store will offer an availability the same as or near enough to that of the users' expectations to update configuration in the Software System.
+    Where an Intermediate State Store is used, Reconciliation is used between the State Store and the Intermediate State Store and then again between the Intermediate State Store and the Software System.
+
 - ## Feedback
 
     Open GitOps follows [control-theory](https://en.wikipedia.org/wiki/Control_theory) and operates in a closed-loop. In control theory, feedback represents how previous attempts to apply a desired state have affected the actual state. For example if the desired state requires more resources than exist in a system, the software agent may make attempts to add resources, to automatically rollback to a previous version, or to send alerts to human operators.
 
+- ## Break the Glass
+    The process of editing the Intermediate State Store directly in the event that a configuration update needs to be made to the Software System but the State Store is unavailable.
```

Comment:
I'd suppose an example of why the state store is not available might be helpful to understand the potential situation better. Also a note that this should really only be a very rare exception *and* that proper authorization must be in place (e.g. multi-party authorization).

Reply:
State store can be anything? s3? cassandra?